cdp-agentkit-core/src/tests/wrap_eth_test.ts

```diff
@@ -0,0 +1,85 @@
+import { Coinbase, ContractInvocation, Wallet } from "@coinbase/coinbase-sdk";
+
+import { WETH_ABI, WETH_ADDRESS, wrapEth, WrapEthInput } from "../actions/cdp/wrap_eth";
+
+const MOCK_AMOUNT_TO_WRAP = "100000000000000000";
+
+describe("Wrap Eth", () => {
+  it("should successfully parse valid input", () => {
+    const validInput = {
+      amountToWrap: MOCK_AMOUNT_TO_WRAP,
+    };
+
+    const result = WrapEthInput.safeParse(validInput);
+
+    expect(result.success).toBe(true);
+    expect(result.data).toEqual(validInput);
+  });
+
+  it("should fail parsing empty input", () => {
+    const emptyInput = {};
+    const result = WrapEthInput.safeParse(emptyInput);
+
+    expect(result.success).toBe(false);
+  });
+});
+
+describe("Wrap Eth Action", () => {
+  const NETWORK_ID = Coinbase.networks.BaseSepolia;
+  const TRANSACTION_HASH = "0xghijkl987654321";
+
+  let mockContractInvocation: jest.Mocked<ContractInvocation>;
+  let mockWallet: jest.Mocked<Wallet>;
+
+  beforeEach(() => {
+    mockWallet = {
+      invokeContract: jest.fn(),
+      getDefaultAddress: jest.fn().mockResolvedValue({
+        getId: jest.fn().mockReturnValue(TRANSACTION_HASH),
+      }),
+      getNetworkId: jest.fn().mockReturnValue(NETWORK_ID),
```

Comment:
These mocks are not required for testing `wrap_eth` action.
```suggestion
```

Reply:
Good catch! Addressed.